x/dualstaking/keeper/delegator_reward.go

```diff
@@ -177,6 +193,11 @@ func (k Keeper) RewardProvidersAndDelegators(ctx sdk.Context, providerAddr sdk.A
 	contributorAddresses, contributorPart := k.specKeeper.GetContributorReward(ctx, chainID)
 	if contributorPart.GT(math.LegacyZeroDec()) {
 		contributorsNum := int64(len(contributorAddresses))
+		// Sanity check - contributorsNum > 0
+		if contributorsNum == 0 {
```

Comment:
this can be zero  think, lets break if in this case

Reply:
Fixed in 884b954aca495468a0fe228ae26115382c5f432f